src/commands/projects.rs

```diff
@@ -19,7 +19,12 @@ pub fn execute<'a>(matches: &ArgMatches<'a>) -> Result<(), Error> {
     let api = Api::current();
     let org = config.get_org(matches)?;
     let mut projects = api.list_organization_projects(&org)?;
-    projects.sort_by_key(|p| (p.team.name.clone(), p.name.clone()));
+    projects.sort_by_key(|p| {
+        (
+            p.team.as_ref().map_or(String::new(), |t| t.name.clone()),
```

Comment:
I wonder why the old code uses `clone`. I think you should avoid allocating here, as this is potentially called multiple times. Maybe `p.team.as_ref().unwrap_or("")` should do the trick? Similarly below.

Reply:
We want to access `p.team.name`, so we cannot do `unwrap_or` in this case.

Reply:
Oh, right! Is there a way to avoid the `clone` though? I don’t see any reason why `sort_by_key` would require owned data.